Modify Rule S5131: Message clarity (#688)
SonarSource/rspec: rules/S5131/message.adoc

=== Message

> Sanitize this user-controlled data before reflecting it.

Author: ```suggestion
Change this code to not reflect unsanitized user-controlled data.
```

-> First baby step

Author: I choose not to go with "untrusted" because I can smell the people saying "I trust this data since it's coming from a service that I wrote myself"
